gnomad/resources/grch37/reference_data.py

```diff
@@ -287,3 +373,48 @@ def get_truth_ht() -> hl.Table:
         .repartition(200, shuffle=False)
         .persist()
     )
+
+
+gtex_rsem = VersionedMatrixTableResource(
+    default_version="v7",
+    versions={
+        "v7": GnomadPublicMatrixTableResource(
+            path="gs://gnomad-public-requester-pays/resources/grch37/gtex_rsem/gtex_rsem_v7.mt",
+            import_func=_import_gtex_rsem,
+            import_args={
+                "gtex_path": "gs://gcp-public-data--gnomad/resources/grch37/gtex/bulk-gex_v7_rna-seq_GTEx_Analysis_2016-01-15_v7_RSEMv1.2.22_transcript_tpm.txt.gz",
+                "meta_path": "gs://gcp-public-data--gnomad/resources/grch37/gtex/annotations_v7_GTEx_v7_Annotations_SampleAttributesDS.txt.gz",
+                "min_partitions": 1000,
+            },
+        ),
+    },
+)
+
+gencode = VersionedTableResource(
+    default_version="v19",
+    versions={
+        "v19": GnomadPublicTableResource(
+            path="gs://gnomad-public-requester-pays/resources/grch37/gencode/gencode.v19.annotation.ht",
+            import_func=_import_gencode,
+            import_args={
+                "gtf_path": "gs://gcp-public-data--gnomad/resources/grch37/gencode/gencode.v19.annotation.gtf.gz",
+                "reference_genome": "GRCh37",
+                "force_bgz": True,
+                "min_partitions": 10,
+            },
+        ),
+    },
+)
+
+gencode_cds = VersionedTableResource(
```

Comment:
remove this resource, and only use the full resource with the added filtering function